doc/python/px-arguments.md

```diff
@@ -101,33 +115,55 @@ fig.show()
 Many more examples of wide-form and messy data input can be found in our [detailed wide-form support documentation](/python/wide-form/).
 
 
-### Input Data as Pandas `DataFrame`s
+## Dataframe Input
+
+The first argument of every `px` function is `data_frame`. If you provide a dataframe as a `px` function's first argument, you can then specify column names as strings from the dataframe as other arguments.
+
+### Supported Dataframes
+
+`px` functions natively support pandas, Polars, and PyArrow dataframes. `px` uses [Narwhals](https://narwhals-dev.github.io/narwhals/) to provide this native dataframe support. Other types of dataframes that are currently supported by Narwhals, for example cuDF and Modin, may also work with `px`.
```

Comment:
Note: Need to make sure we are consistent with messaging about whether Modin is supported (it's included in the list of supported DataFrame types in `doc/python/performance.md`)

Reply:
Good catch. I'd changed it in one but not the other. Thanks!